FV3/io/FV3GFS_io.F90

```diff
@@ -578,42 +587,64 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain)
       nullify(var2_p)
     endif
 
-    !--- read the orography restart/data
-    call mpp_error(NOTE,'reading topographic/orographic information from INPUT/oro_data.tile*.nc')
-    call restore_state(Oro_restart)
+    !--- do orography
+    fname = 'INPUT/'//trim(fn_oro)
+    if (file_exist(fname)) then
```

Comment:
You're changing the program logic here.  If the orography files are not found, you are skipping the check which would have stopped the simulation otherwise.  It's a handy thing to have in case one forgets to add the required input files when first setting up a run.